packages/aws-cdk/lib/build.ts

```diff
@@ -0,0 +1,23 @@
+import * as cxapi from '@aws-cdk/cx-api';
+
+type Options = {
+  buildStackAssets: (stack: cxapi.CloudFormationStackArtifact) => Promise<void>;
+};
+
+export async function buildAllStackAssets(stacks: cxapi.CloudFormationStackArtifact[], options: Options): Promise<void> {
```

Comment:
This function used to handle `--concurrency`, but after #21661 reverted `--concurrency`, we have to build sequentially. As-is, this logic could arguably be moved into `CdkToolkit`. But, I wanted to wait until I get more feedback on whether this PR is going in the right direction before I make too many changes.

Reply:
`--concurrency` is being re-rolled, but I'm honestly not quite sure if building in parallel makes sense. The logs will probably intersperse in unreadable ways if we do that.